README.md

```diff
@@ -273,7 +282,18 @@ $ pie install xdebug --not-a-defined-configuration-option # this would fail
 
 ### Windows installation
 
-Determine the expected name for the Windows DLL:
+When installing the extension on Windows, the PHP version invoking PIE would be used, unless `--with-php-path=<php-path>` is
+provided. Example:
+
+```powershell
+# Assuming C:\usr\php8.3.4\php.exe is in the $PATH
+$ pie install xdebug                                     # uses C:\usr\php8.3.4\php.exe (i.e. the version that invoked PIE)
+$ pie install xdebug --with-php-path="C:\usr\php7.4.33"  # uses C:\usr\php7.4.33\php.exe
```

Comment:
nit: These lines are too long, they even wrap in the GH review pane.

Reply:
Will reflow the whole MD doc to 80 characters after merge :+1: